>
```[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-944)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-945)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-946)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-947)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-948)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-949)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-950)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-951)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-952)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-953)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-954)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-955)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-956)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-957)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-958)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-959)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-960)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-961)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-962)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-963)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-964)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-965)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-966)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-967)[](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.22/detail/mini-e2e-helm_k8s-1.22/2414/pipeline#step-106-log-968)• Failure [1480.011 seconds]

RBD

/go/src/github.com/ceph/ceph-csi/e2e/rbd.go:223

  Test RBD CSI

  /go/src/github.com/ceph/ceph-csi/e2e/rbd.go:370

    Test RBD CSI [It]

    /go/src/github.com/ceph/ceph-csi/e2e/rbd.go:371


    Mar 31 12:07:16.844: timed out waiting for the rbd-nbd process: rbd-nbd process is not running yet: command terminated with exit code 1


    /go/src/github.com/ceph/ceph-csi/e2e/rbd.go:1406

------------------------------



Summarizing 1 Failure:


[Fail] RBD Test RBD CSI [It] Test RBD CSI 

/go/src/github.com/ceph/ceph-csi/e2e/rbd.go:1406


Ran 2 of 4 Specs in 2388.353 seconds

FAIL! -- 1 Passed | 1 Failed | 0 Pending | 2 Skipped

--- FAIL: TestE2E (2388.36s)

FAIL

make: *** [Makefile:185: run-e2e] Error 1

script returned exit code 2
``` 

@nixpanic looks like the image pull error is gone :+1: , and its failing now with above

--- a/build.env
+++ b/build.env
@@ -12,8 +12,8 @@
 CSI_IMAGE_VERSION=canary
 
 # Ceph version to use
-BASE_IMAGE=quay.io/ceph/ceph:v16
-CEPH_VERSION=pacific
+BASE_IMAGE=quay.io/ceph/ceph:v17
+CEPH_VERSION=quincy
 
 # standard Golang options
 GOLANG_VERSION=1.17.5

